learn/teams/manage-subscription.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Manage Subscription
+id: ""
+---
+---
+
+You can delegate subscription administration to manage members, view the current subscription, members and your subscription expiration date.
+
+-  You can access the Manage Subscription page in the administration area by clicking on the **Manage Subscription** entry in the left panel.
+
+![Manage Subscription](/learn/assets/TeamManageSubscription.png)
+
+
+-  To Change the owner of subscription, click on **Change Owner** button, a pop-up will open, select the member to be made new subscription owner and click on **Save** button.
```

Comment:
What is the use case for change owner ?
Diffirentiage  between wavemaker users vs own team members